packages/mock-block-dock/src/debug-view/dev-tools/datastore-graph-visualisation.tsx

```diff
@@ -0,0 +1,211 @@
+import { Entity, Link } from "@blockprotocol/graph/.";
+import { Box } from "@mui/material";
+import { GraphChart, GraphSeriesOption } from "echarts/charts";
+import * as echarts from "echarts/core";
+import { SVGRenderer } from "echarts/renderers";
+import { useEffect, useRef, useState } from "react";
+
+import { useMockBlockDockContext } from "../../mock-block-dock-context";
+
+const parseNameFromEntity = (entity: Entity): string =>
```

Comment:
We have a newish schema property in the BP, `labelProperty`, which allows users to specify which field on an entity should be used as its display name. We have this set on the sample entities in MBD so we should use it here, with some sensible fallbacks.

I've more or less lifted the approach in HASH in [this commit](https://github.com/blockprotocol/blockprotocol/commit/1538378979a2ac693fa07f1c0d5dda0f0202890e) as an example

Reply:
Thanks Ciaran, missed that this made it into MBD already - have cherry-picked your commit in https://github.com/blockprotocol/blockprotocol/pull/605/commits/d01e699d3b7fa7c6add939d174f7078569831065 👍